permissions/plugin-pipeline-gcp.yml

```diff
@@ -0,0 +1,11 @@
+---
+name: "pipeline-gcp"
+github: &GH "jenkinsci/pipeline-gcp-plugin"
+issues:
+  - github: *GH
+paths:
+  - "org/jenkins-ci/plugins/pipeline-gcp"
+developers:
+  - "enigo"
+cd:
+  enabled: true
```

Comment:
```suggestion
```
You can enable that once your plugin is prepared for CD.

Reply:
but that's what this [PR](https://github.com/jenkinsci/pipeline-gcp-plugin/pull/3) is about, I thought

Reply:
Nope, this just adds upload permissions to artifactory

Reply:
but I added [workflow/cd.yaml](https://github.com/jenkinsci/pipeline-gcp-plugin/pull/3/files#diff-7820634de9167dbbf6a1ab9e9c02aab3b8b0ca512494aca230074fd13f630ade) there as per the instructions :thinking:

Reply:
There's more to do and PRs are reviewed separately

Reply:
yeah, I added\removed other things as well
ok, will open a separate PR then, no probs